india_compliance/gst_india/report/api_requests/api_requests.js

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2025, Resilient Tech and contributors
+// For license information, please see license.txt
+
+frappe.query_reports["API Requests"] = {
```

Comment:
This report can be added to workspace for quick access.
Add screenshots so that others can know about this report (as it's not added to docs)